Makefile.am

```diff
@@ -829,14 +829,24 @@ ruby/bin/ronn:
 	if [ ! -f $@ ]; then pushd ruby/bin && ln -sf ronn.ruby* ronn && popd; fi
 
 $(ronn_man): man/%: ruby/bin/ronn ronn/%.ronn ronn/index.txt ronn/header.txt
-	mkdir -p man
-	export GEM_PATH=`readlink -f $(top_srcdir)/ruby`:$$GEM_PATH && \
-	cd ronn && \
-	../ruby/bin/ronn --date=`date "+%Y-%m-%d"` \
+	TARGET_DOC=`basename $@` ; \
+	RONN_DIR="$(abs_srcdir)/ronn" ; \
+	if [ ! "$(abs_srcdir)/ronn" -ef "$(abs_top_builddir)/ronn" && $$TARGET_DOC != "geopm.7" ] ; then \
```

Comment:
That isn't how and works in bash:

https://stackoverflow.com/questions/13408493/an-and-operator-for-an-if-statement-in-bash

I literally wrote the solution in a comment you ignored:

https://github.com/geopm/geopm/pull/1257#discussion_r474967162